portal-api/src/graphql/schemas/index.js

```diff
@@ -52,6 +51,7 @@ type Deal {
   _id: String!
   dealType: String
   submissionType: String
+  updatedAt: Float
```

Comment:
Float or Integer, if decimal is not required. If required please ignore.

Reply:
ATM sometimes it generates a decimal depending on the time